pkg/server/nodemanager/node_scaler.go

```diff
@@ -81,6 +87,10 @@ func (s *BindingNodeScaler) createNodeForPod(pod *api.Pod) *api.Node {
 	if pod.Spec.Spot.Policy == api.SpotAlways {
 		isSpotPod = true
 	}
+	isDedicatedPod := false
+	if pod.Spec.Dedicated {
+		isDedicatedPod = true
+	}
```

Comment:
Why not `isDedicated := pod.Spec.Dedicated`?

Reply:
simplified it further just setting node.Spec.Dedicated equal to pod.Spec.Dedicated